internal/keyvault/authorizer.go

```diff
@@ -0,0 +1,67 @@
+// Copyright 2020 Google LLC
```

Comment:
Nit: could we name this `azurekeyvault` just to avoid collisions?

Reply:
sure thing